Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingFragmentViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package de.rki.coronawarnapp.ui.onboarding
+
+import com.squareup.inject.assisted.AssistedInject
+import de.rki.coronawarnapp.ui.SingleLiveEvent
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModel
+import de.rki.coronawarnapp.util.viewmodel.SimpleCWAViewModelFactory
+
+class OnboardingFragmentViewModel @AssistedInject constructor() : CWAViewModel() {
+    val routeToScreen: SingleLiveEvent<OnboardingNavigationEvents> = SingleLiveEvent()
+
+    fun onNextButtonClick() {
+        routeToScreen.value = OnboardingNavigationEvents.NavigateToOnboardingPrivacy
```

Comment:
Please use `postValue(...)`, which is async and means there are less requirements on the caller (which thread it is being called from).